src/sky.py

```diff
@@ -32,19 +35,18 @@ def get_color(self):
 
         # find nearest hours in self.colors
         color_index = 0
-        for index in range(len(self.colors)):
-            color_hour = int(list(self.colors.keys())[index])
+        for index, color_hour in enumerate(self.colors_hours):
             if precise_hour < color_hour:
                 color_index = index - 1
                 break
         else:
             color_index = -1
 
         # start and end colors
-        start_color = list(self.colors.values())[color_index]
-        end_color = list(self.colors.values())[color_index + 1]
-        start_hour = int(list(self.colors.keys())[color_index])
-        end_hour = int(list(self.colors.keys())[color_index + 1])
+        start_color = list(self.colors_rgb)[color_index]
+        end_color = list(self.colors_rgb)[color_index + 1]
+        start_hour = int(self.colors_hours)[color_index]
+        end_hour = int(self.colors_hours)[color_index + 1]
```

Comment:
Slight oopsie here: you can't subscript an int. You might have intended to do this instead:

```py
start_hour = int(self.colors_hours[color_index])
end_hour = int(self.colors_hours[color_index + 1])
```

Reply:
I think you reviewed outdated commits

Reply:
I already solved these problems in the 'Bug fix' commit

Reply:
I just saw you pushed a commit yes